packages/api-reference/src/hooks/useReactiveSpec.ts

```diff
@@ -1,7 +1,8 @@
-import { fetchSpecFromUrl, parse } from '@scalar/swagger-parser'
+import { fetchSpecFromUrl } from '@scalar/swagger-editor'
```

Comment:
Let's either add this to the parser, create an @scalar/utils package or move it to references for now.